.openpublishing.redirection.json

```diff
@@ -18,6 +18,14 @@
         "redirect_url": "/dotnet/articles/core/tools/dotnet-test"
     },
     {
+        "source_path": "docs/csharp/csharp-6.md", 
+        "redirect_url": "/dotnet/articles/csharp/whats-new/csharp-6"
```

Comment:
Added this flag to the entries for whats-new/index, whats-new/csharp-6, and whats-new/csharp-7.